docs/process/sync-testing.md

```diff
@@ -0,0 +1,139 @@
+
+1. First, you'll need to [setup Lockwise for Android locally](https://github.com/mozilla-lockwise/lockwise-android/blob/master/docs/install.md). Make sure to at least open the project in [Android Studio](https://developer.android.com/studio/install) once, which will force it to install the correct dependencies.
+2. Now, open [AccountStore.kt](https://github.com/mozilla-lockwise/lockwise-android/blob/master/app/src/main/java/mozilla/lockbox/store/AccountStore.kt) and change the [config declaration](https://github.com/mozilla-lockwise/lockwise-android/blob/master/app/src/main/java/mozilla/lockbox/store/AccountStore.kt#L269) from `ServerConfig.release` to `ServerConfig.dev`.
+3. Rebuild the project in Android Studio (Build > Make Project).
+4. Now, open the [Android Virtual Device Manager](https://developer.android.com/studio/run/managing-avds) within Android Studio, and make sure you have a device downloaded.
+5. Open your device in the [Android Studio Emulator](https://developer.android.com/studio/run/emulator), and login using an existing [staging Firefox account](/ecosystem-platform/docs/process/using-the-staging-environment#working-with-staging-firefox-accounts). If you've previously logged in using this virtual device while pointed to production, you may need to download a new device to bypass auth caching.
+
+
+## How do I...
+
+## ...find my Sync id?
+
+Each Firefox account has a unique Sync id associated with it. There's a few different ways to find it:
+
+#### via the Browser Console:
+
+1. First, make sure that the Browser Console (different than Developer Tools Console) is enabled. Open the [Developer Tools Settings](https://developer.mozilla.org/en-US/docs/Tools/Settings) (Desktop menu: Tools > Web Developer > Toggle Tools > “…” > Settings), and check the “Enable browser chrome and add-on debugging toolboxes” setting in the “Advanced settings” section.
+
+2. In `about:config`, set `devtools.chrome.enabled` to `true`. This will allow you to type into the Browser Console.
+
+3. Now, manually Sync (Account Menu > Sync Now).
+
+4. Open the Browser Console (Tools > Web Developer > Browser Console) and type `Weave.Service.clusterURL`. 
+
+   You should see something similar to `https://sync-3-us-east-1.stage.mozaws.net/1.5/0000000/`, where `0000000` corresponds to your Sync Id.
+
+#### via Sync Logs:
+
+1. First, verify that you've configured your Sync logs to [log successful Syncs](#configure-sync-logging).
+
+2. Now, manually Sync (Account Menu > Sync Now).
+
+3. Now, [browse to your Sync logs](#configure-sync-logging), and open the most recent one. Scan through it for something that looks like this: `POST success 200 https://sync-3-us-east-1.stage.mozaws.net/1.5/0000000/storage/tabs?batch=true&commit=true`. The `0000000` will correspond to your Sync Id.
+
```

Comment:
Not sure if you want to also mention the [About Sync extension](https://addons.mozilla.org/en-US/firefox/addon/about-sync/)?

Reply:
oooh, yes. Good call. I'll update, and add it.

Reply:
@jrconlin welp, I went to go add it, and looks like I already [had, here](https://github.com/mozilla/ecosystem-platform/pull/52/files#diff-ee61870c4762e9d6128498ad87c3f90dR126) - I figured sticking it under configuring logging made sense, but if you think there's a place for it that makes more sense, I'm all 👂

Reply:
Ah! No, that's fine. I just missed it.